cadquery/assembly.py

```diff
@@ -436,6 +437,7 @@ def save(
         self,
         path: str,
         exportType: Optional[ExportLiterals] = None,
+        exportMode: Optional[STEPExportModeLiterals] = None,
```

Comment:
No need to make this optional, just set a backward compatible default

Reply:
Ok, this one simplifies the fix for the comment above a lot. I'll make that change.